.gitignore

```diff
@@ -1,6 +1,5 @@
 node_modules
 npm-debug.log
 .DS_Store
-lib
```

Comment:
Removed the lib from gitignore.  There are build issues when trying to pull this into the FE repo .  The lib isn't being properly built when being included in the FE, so including the lib here is a quick fix for this issue.